src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/resource/ReportResource.kt

```diff
@@ -84,10 +84,29 @@ class ReportResource(
     ],
   )
   fun getBasicReports(
+    @Parameter(
+      description = "Filter by given locations, typically prison IDs",
+      example = "LEI,MDI",
+      array = ArraySchema(
+        schema = Schema(example = "MDI", minLength = 2, maxLength = 20),
+        arraySchema = Schema(
+          requiredMode = Schema.RequiredMode.NOT_REQUIRED,
+          nullable = true,
+          defaultValue = "null",
+        ),
+      ),
+    )
+    @RequestParam(required = false)
+    location: List<
+      @Size(min = 2, max = 20)
```

Comment:
why must there be at least 2?

Reply:
this annotation does not affect the size of the list of filters